app/services/doc_auth_router.rb

```diff
@@ -199,6 +199,17 @@ def self.client(warn_notifier: nil)
   # rubocop:enable Layout/LineLength
 
   def self.doc_auth_vendor
+    if IdentityConfig.store.doc_auth_vendor_randomize
+      target_percent = IdentityConfig.store.doc_auth_vendor_randomize_percent
+
+      target_percent = target_percent > 100 ? 100 : target_percent
+      target_percent = target_percent < 0 ? 0 : target_percent
+
+      if rand(100) < target_percent
+        return IdentityConfig.store.doc_auth_vendor_randomize_alternate_vendor
+      end
+    end
+
     IdentityConfig.store.doc_auth_vendor
```

Comment:
can we input the user uuid and hash that? so we have something deterministic? otherwise this could flip clip for the same user in the middle of proofing